resolvers/webpack/index.js

```diff
@@ -65,7 +65,11 @@ exports.resolve = function (source, file, settings) {
 
       log('Config path resolved to:', configPath)
       if (configPath) {
-        webpackConfig = require(configPath)
+        try {
+          webpackConfig = require(configPath)
+        } catch(e) {
+          console.log('Error resolving webpackConfig', e)
```

Comment:
I think we should also rethrow the error; that way code execution continues to stop.